src/Stratis.Sidechains.Networks/FederatedPegMain.cs

```diff
@@ -34,19 +34,12 @@ internal FederatedPegMain()
 
             this.Genesis = genesisBlock;
 
-            // Keeping the 3rd there in case we use it in future. For our integration tests we use 2 nodes currently.
-            this.FederationKeys = new Key[]
-            {
-                new Mnemonic("lava frown leave wedding virtual ghost sibling able mammal liar govern wisdom").DeriveExtKey().PrivateKey,
-                new Mnemonic("idle power swim wash diesel blouse photo among eager reward wide hair").DeriveExtKey().PrivateKey,
-                new Mnemonic("high neither night category fly wasp inner kitchen phone current skate menu").DeriveExtKey().PrivateKey
-            };
-
+            //todo : replace dummy values before release
```

Comment:
I liked being able to see where these pub keys are coming. Is there a reason for this change?

Reply:
yes for the release you want to keep these secret otherwise anybody can produce blocks, not only POA members right ?